### PR TITLE
Removing 2 work lines

### PR DESCRIPTION
**Pull request by cyberscorpion**
This is the description of pull request. We are removing unused code.

### CODE REVIEW

File: newfile.txt [hunk 1]
```diff
@@ -2,5 +2,3 @@ To create new branch: `git checkout -b test`
 to swifth branch: `git checkout <branchName>
 To merge branch: `git merge <branchName>` (merges another branch to current br.)
 git log --oneline
-Work1
```
Comment (cyberscorpion) [inline newfile.txt:5]: Good work!

File: newfile.txt [hunk 2]
```diff
@@ -2,5 +2,3 @@ To create new branch: `git checkout -b test`
...
-work2
```
Comment (manishdipu) [inline newfile.txt:6]: Good. Work

File: newfile.txt [hunk 3]
```diff
@@ -2,5 +2,3 @@ To create new branch: `git checkout -b test`
 to swifth branch: `git checkout <branchName>
 To merge branch: `git merge <branchName>` (merges another branch to current br.)
 git log --oneline
```
Comment (RachitJain1999) [inline newfile.txt]: LGTM!

**Comment by adityagangadharakuballi** [comment]
Good work

**Pull request merged**